android/player/BUILD

```diff
@@ -26,13 +26,20 @@ distribution(
     maven_coordinates = "com.intuit.playerui:android:%s" % VERSION,
 )
 
-kt_jvm_junit5_test(
+kt_player_test(
     name = "player-tests",
-    srcs = glob(["src/test/java/**"]),
-    associates = [":player-kotlin"],
-    kotlinc_opts = "//jvm:test_options",
-    test_package = "com.intuit.playerui.android",
-    deps = [":player"] + test_deps,
+    srcs = [
+        "src/test/java/com/intuit/playerui/android/AndroidPlayerTest.kt",
+        "src/test/java/com/intuit/playerui/android/AssetContextTest.kt",
+        "src/test/java/com/intuit/playerui/android/BrokenAssetTest.kt"
```

Comment:
the first two are not great because the test target should really be in the test package, but it's moved here because of the associates line below, allowing the tests to access internal player API.

the `BrokenAssetTest` is _really_ bad since i had to move it out of the original package, otherwise i couldn't reference it as a source. It's really the same problem trying to access internal API

the main problem here is `:player-kotlin` target is created as an intermediate target by the db android macro, and the visibility isn't marked as public so it can only be used from this package. The end result target does not meet the requirements to be used as an associate

I could either modify the grab rules or we need to write some hack, maybe wrapper library to expose `:player-kotlin`? Is there a better way